detection-rules/attachment_fake_attachment_image.yml

```diff
@@ -53,10 +53,7 @@ source: |
     )
     or sender.email.domain.root_domain not in $high_trust_sender_root_domains
   )
-  and (
-    not profile.by_sender().solicited
-    or profile.by_sender().any_messages_malicious_or_spam
-  )
+  and profile.by_sender().any_messages_malicious_or_spam
```

Comment:
This would seem to require a previous malicious classification from the same sender. Do you think about removing all inclusionary sender profile conditions and use `and not profile.by_sender_email().any_messages_benign`?